### ISSUE TITLE
Compression doesn't work right

### ISSUE THREAD

**Issue by mikeTWC1984** [aloneguid/parquet-dotnet#130]
I noticed that Parquet.net parquet files are not much better than compressed CSV. I expected much better result because of column level compression.
I did a dumb experiment - generated 100k and 200k row tables repeating same exact row. Turns out Parquet.Net generated files grows linearly with row count, although it supposed to be about static (and I can achieve that with pyarrow for exmple). More over simple csv.zip file is 10X smaller.

rows   |  Raw CSV  | Parquet.net | CSV.zip
100K   |   14.3 M    |   674 K        |  59KB
200K   |   28.5 M    |  1345K        | 118 K

Questions:
 Does Parquet.Net compress columns, rows or entire file?
 Does Parquet.Net include some hidden metadata column/s? (how to explain linear file growth?)

**Comment by mikeTWC1984** [comment]
I opened generated parquet file in text editor, it contains a lot of repetitive sequences on NULL and some other special characters. If I zip that parquet file it goes down to few KB (expected size). I also tried gzip - the file size is better (~ 100KB for 100k rows, but still far from few KB).
I also checked file metadata in pyarrow - it's showing that file has 1 row group (I thought it might have bunch of  small groups, but it's not the case)
